docs/release-notes/artifacts/pr0179.yaml

```
@@ -0,0 +1,15 @@
+# --- Release notes artifact ----
+
+schema_version: 1
+changes:
+  - title: fix typo
```

Since the `title` is very important for `bugfix` entries (as the script only uses the title in the rendered output), I think we should be more descriptive here. Something like:
```suggestion
  - title: Fixed a typo in the cookie fetch method in the load-balancing algorithm
```